src/components/Main/MainView/MessagesView.vue

```diff
@@ -77,25 +77,21 @@ const useLoadMessages = () => {
     ),
     isInitialLoad: computed(() => store.state.domain.messagesView.isInitialLoad)
   })
-  const loadFormerMessages = async () => {
+  const loadMessages = async (direction: 'former' | 'latter') => async () => {
```

Comment:
```suggestion
  const loadMessages = (direction: 'former' | 'latter') => async () => {
```
ここに`async`つけると戻り値の型が`Promise<() => Promise<void>>`になってうまく動かなかったです